README.md

```diff
@@ -23,6 +23,8 @@ To bind the java agent to a specific IP change the port number to `host:port`.
 
 See `./run_sample_httpserver.sh` for a sample script that runs the httpserver against itself.
 
+Note: Requires minimum JDK version 1.6
```

Comment:
I'm not sure this needs to be mentioned here, even 6 is fairly old by now.

Reply:
Removed the note and updated PR.